flow/scripts/deletePowerNets.tcl

```diff
@@ -1,29 +0,0 @@
-read_lef $::env(TECH_LEF)
-read_lef $::env(SC_LEF)
-if {[info exist ::env(ADDITIONAL_LEFS)]} {
-  foreach lef $::env(ADDITIONAL_LEFS) {
-    read_lef $lef
-  }
-}
-
-# Read liberty files
-foreach libFile $::env(LIB_FILES) {
-  read_liberty $libFile
-}
-# Read def and sdc
-read_def $::env(RESULTS_DIR)/6_final.def
-
-proc deleteNetByName {name} {
-  set db [ord::get_db]
-  set chip [$db getChip]
-  set block [$chip getBlock]
-  set net [$block findNet $name]
-  $net destroySWires
-  puts "\[INFO\] Deleted net '[$net getName]'"
-}
-
-deleteNetByName VDD
-deleteNetByName VSS
```

Comment:
@maliberty do you know if the GUI is set to hide power nets, will they still appear in the saved image? Also, is there such a command available in Tcl?

Reply:
gui::set_display_controls "Nets/Power" visible false
    gui::set_display_controls "Nets/Ground" visible false

the image should respect the controls.

Reply:
@vvbandeira you can also use the `-display_option` on `save_image` to control the settings specifically for the image